logiclearner/main/views.py

```diff
@@ -30,6 +32,16 @@ class HintApiView(APIView):
     def dispatch(self, *args, **kwargs):
         return super(HintApiView, self).dispatch(*args, **kwargs)
 
+    def post(self, request):
+        next_expr = request.data.get('next_expr')
```

Comment:
Always a good idea to specify a default value for a `get` to prevent any inadvertent 500 errors.

Reply:
Got it!